doc/source/reference/arrays.rst

```diff
@@ -60,6 +60,36 @@ is an :class:`ArrowDtype`.
 `Pyarrow <https://arrow.apache.org/docs/python/index.html>`__ provides similar array and `data type <https://arrow.apache.org/docs/python/api/datatypes.html>`__
 support as NumPy including first-class nullability support for all data types, immutability and more.
 
+The table below shows the equivalent pyarrow-backed (``pa``), pandas extension, and numpy (``np``) types that are recognized by pandas.
+
+===================================== ========================== ===================
+Pyarrow type                          pandas extension type      Numpy type
+===================================== ========================== ===================
+``pd.ArroeDtype(pa.bool_())``         :class:`BooleanDtype`      ``np.bool_``
```

Comment:
Do you think repeating `pd.ArrowDtype` to wrap every arrow type in this column adds value? Since the column title is `Pyarrow type`, feels like just using `pa.bool_()`... is good enough. And maybe a note to comment on creating a pandas arrow type before or after the table could be useful.

Reply:
Fair point. I'll link link to the pyarrow factory type functions and explain they need to be passed into `pd.ArrowDtype` explicitly